### ISSUE TITLE
Cryptography builds fail

### ISSUE THREAD

**Issue by harkylton** [antocuni/pypy-wheels#22]
First of all, thanks for providing these wheels. It will reduce out build times significantly.

The builds for cryptography are failing on PyPy 7.3.1. I'm guessing it has something to do with the fact that they dropped support for OpenSSL 1.0.2g in cryptography 3.2, but I'm not sure.

**Comment by antocuni** [comment]
yeah, I see that they fails but didn't find the time to investigate/fix. I will be happy to give you commit rights if you feel like helping
